env/staging/elasticache/terragrunt.hcl

```
@@ -37,6 +37,7 @@ inputs = {
   elasticache_node_count    = 1
   elasticache_node_type     = "cache.t3.micro"
   vpc_private_subnets       = dependency.common.outputs.vpc_private_subnets
+  sns_alert_warning_arn     = ""
```

```suggestion
  sns_alert_warning_arn     = dependency.common.outputs.sns_alert_warning_arn
```

Should be set to this value